### DISCUSSION TITLE
A Room to the Far, Far North

### DISCUSSION THREAD

**Discussion by JeremyBickel** [endless-sky/endless-sky#8216] [Q&A] [answered]
When I first started playing, years ago, I began in a room to the far north, like in another galaxy, and there was a one-way exit to the main map. Has anyone else seen that?

**Comment by Zitchas** [comment]
Sorry, no, I haven't seen that.

**Comment by movingpictures** [comment]
IIRC, it was a (badly broken) plugin.

**Reply by JeremyBickel** [reply]
Ha! I just spent some time trying to find the IIRC plugin, then I found the acronym's meaning. Do you happen to remember the name of the plugin? It was tiny, but I thought it had a cool idea (IIRC).

**Comment by movingpictures** [comment]
Sadly,  no. It wasn't functional by the time I found it two or so years ago.

**Comment by RisingLeaf** [comment]
You searched through the es plugin archive?

**Comment by JeremyBickel** [comment]
I've looked around but haven't found it. If I remember, it's location is probably in the tens of thousands north, and it just had a few rooms with a portal to the main map. I suppose it doesn't matter much, because it was tiny, but I just remember liking those few rooms ..did it have a highly polished metallic and white android feel (not that androids were involved)? <shrug> I just don't remember.

**Comment by Hecter94** [comment, marked as answer]
It sounds somewhat similar to early versions of Omnis. 
You had a ringworld with a couple of landable spots where you could purchase most of the available ships/outfits, get credits, licenses etc. 
Then it had a wormhole back to the main galaxy.

**Comment by JeremyBickel** [comment]
Hey, I think that could be it. The location looks right, if (0, -10000) is due north. Thanks!
